scss/product/components/_badge.scss

```diff
@@ -59,6 +59,10 @@
   height: 12px;
   width: 12px;
   vertical-align: middle;
+  &.circle-small{
```

Comment:
And circle-small?

Reply:
badge-sm class can be used here, i will remove this style